sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -65,19 +69,58 @@ class FlinkRunnerTest(portable_runner_test.PortableRunnerTest):
     _use_grpc = True
     _use_subprocesses = True
 
+    conf_dir = None
+
+    @classmethod
+    def tearDownClass(cls):
+      if cls.conf_dir and exists(cls.conf_dir):
+        logging.info("removing conf dir: %s" % cls.conf_dir)
+        rmtree(cls.conf_dir)
+      super(FlinkRunnerTest, cls).tearDownClass()
+
+    @classmethod
+    def _create_conf_dir(cls):
+      """Create (and save a static reference to) a "conf dir", used to provide
```

Comment:
This looks convoluted. Could we just check in the file and point to that directory instead?

Reply:
good idea, though I am also using this directory as the tempdir that holds the output metrics; I think I'd still need to do that and it would look the same as this, but perhaps with a different name.

Reply:
makes sense